server/service/src/requisition_line/response_requisition_line/update.rs

```diff
@@ -144,7 +144,7 @@ fn generate(
         u.addition_in_units = updated_addition_in_units.unwrap_or(u.addition_in_units);
         u.expiring_units = updated_expiring_units.unwrap_or(u.expiring_units);
         u.days_out_of_stock = updated_days_out_of_stock.unwrap_or(u.days_out_of_stock);
-        u.option_id = updated_option_id.or(u.option_id);
+        u.option_id = updated_option_id;
```

Comment:
Assuming you changed this so its clearable? I think this way okay for now, seeing as we know we always send the full draft in our update mutations, but I think this should be NullableUpdate, so that if we send `UpdateInput { expiring_units: 10 }` it doesn't clear the reason 😁

Reply:
The draft keeps the reason, so it doesn't clear unless the draft is updated to null

Reply:
Yep - just mean if we were to use this API from somewhere else!

Reply:
Oh yes definitely!